app/src/types/user.js

```diff
@@ -14,15 +14,17 @@ const UserModel = t.object(
   t.property("spid_idp", t.string()),
   t.property("fiscal_code", t.string()),
   t.property("name", t.string()),
-  t.property("family_name", t.string())
+  t.property("family_name", t.string()),
+  t.property("preferred_email", t.string())
 );
 
 const SpidUserModel = t.object(
   t.property("fiscalNumber", t.string()),
   t.property("name", t.string()),
   t.property("familyName", t.string()),
   t.property("sessionIndex", t.string()),
-  t.property("issuer", t.any())
+  t.property("issuer", t.any()),
+  t.property("email", t.any())
```

Comment:
why `any` and not `string` (same question for `issuer`)

Reply:
email should be `string`. issuer is an object and needs a more complex validation, we could fix this in https://www.pivotaltracker.com/story/show/154134211

Reply:
do we need/use the value of `issuer`?

Reply:
yes we need it because it contains the IDP chosen for the login. We need this value to generate the logout assertion

Reply:
what is the structure of the object? if possible I suggest to use a slightly more specific definition for the object instead of `any`... we can just include the fields we need, by default `io-ts` ignores extra fields

Reply:
ops, sorry I didn't catch the change :) anyhow, it looks equivalent, thanks